tests/unit/customizations/configure/test_sso.py

```diff
@@ -49,8 +49,11 @@ def test_user_hits_enter_returns_current(self):
         self.assertEqual(response, 'default_value')
 
     def assert_expected_completions(self, completions):
+        # The order of the completion list can vary becuase it comes from the
+        # dict's keys. Asserting that each expected completion is in the list
         _, kwargs = self.mock_prompter.call_args_list[0]
-        self.assertEqual(kwargs['completer'].words, completions)
+        for completion in completions:
```

Comment:
We should also assert the length for the completions or I believe unittest has `assertItemsEqual` (Python 2) and `assertCountEqual` (Python 3) that we can use.